applications/app/controllers/AtomPageController.scala

```diff
@@ -110,29 +108,6 @@ class AtomPageController(
   ): Action[AnyContent] =
     Action.async { implicit request =>
       lookup(s"atom/$atomType/$id") map {
-        case Right(atom: AudioAtom) => {
-
-          /*
-          mark: 57cadc98-16c0-49ac-8bba-c96144c488a7
```

Comment:
There is [another file with this mark](https://github.com/guardian/frontend/blob/c354525345da140b0c596e32de130c215bc2b621/common/app/views/fragments/atoms/audio.scala.html#L4). Should it be removed as well?

Reply:
Good catch! I deleted this file, it is not used anywhere